movieApp/styles.css

```diff
@@ -1,17 +1,53 @@
+*{
+    padding:0;
+    margin:0;
+    text-decoration: none;
+    box-sizing: border-box;
+}
 
 body{
-    background-color: cadetblue;
+    font-family:Georgia, 'Times New Roman', Times, serif ;
+    background-color: black;
 }
 
+nav{
+    background-color: blueviolet;
+    height: 90px;
+    width: 100%;
+}
 
-footer{
-    background-color: black;
-    width: max-content;
-    height: 100px;
+label.logo{
+    color: rgb(243, 96, 145);
+    font-size: 30px;
+    font-weight: bold;
+    line-height: 50px;
 }
 
-#banner{
-height: 500px;
- background-color: rgb(168, 145, 145); 
- text-align: center;
-}
\ No newline at end of file
+label.logoText{
+    color:white ;
+    font-size: 10px;
+    }
```

Comment:
pay attention for indentation. As well you can install [Prettier plugin](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) for VSCode and he'll take care of those things!